ci/README.md

```diff
@@ -77,8 +77,8 @@ The build artifact can be found in `/tmp/envoy-docker-build/envoy/source/exe/env
 
 The `./ci/run_envoy_docker.sh './ci/do_ci.sh <TARGET>'` targets are:
 
-* `bazel.api` &mdash; build and run API tests under `-c fastbuild` with clang-6.0.
```

Comment:
can we remove the versions from here except the first one (L29)

Reply:
Good call, done.